wifiphisher/pywifiphisher.py

```diff
@@ -237,6 +246,18 @@ def targeting_cb(pkt):
     global APs, count
 
     bssid = pkt[Dot11].addr3
+    rssi  = -100
+    # Show signal power
+    if pkt.type == 0 and pkt.subtype == 8:
+        if pkt.haslayer(Dot11Beacon) or pkt.haslayer(Dot11ProbeResp):
+            try:
+                extra = pkt.notdecoded
+                rssi = -(256 - ord(extra[-4:-3]))
+
+            except:
+                rssi = -500
```

Comment:
Can you tell us a few more things about this? What is this exception is handling exactly?